### PR TITLE
add ModernDive

### PR DESCRIPTION
**Pull request by rudeboybert**
moderndive

### CODE REVIEW

**Comment by Ruxton** [comment]
Please update your description to something useful that describes ModernDive.

Also, please update your order_online_url to be a direct link to the sticker in the market place.

**Comment by rudeboybert** [comment]
Hello, 
- updated the `description`
- `order_online_url` is left blank. I contacted stickermule to ask if they could add our design, to which they replied: *Please note that we are re-evaluating the marketplace's future viability and, consequently, not accepting new designs. :(*

File: meta/moderndive.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "moderndive",
+  "author": "Chester Ismay and Albert Y. Kim",
+  "license": "CC0",
+  "raster": "http://hexb.in/hexagons/ModernDive.png",
+  "vector": "http://hexb.in/vector/ModernDive.ai",
+  "description": "An open-source and fully-reproducible electronic textbook bridging the gap between traditional introductory statistics and data science courses available at moderndive.com",
+  "order_online_url": ""
```
Comment (Ruxton) [inline meta/moderndive.json]: Remove the order_online_url if it is going to be empty.

**Review by Ruxton — APPROVED** [review]

**Review by Ruxton — CHANGES_REQUESTED** [review]
Please remove the order_online_url

**Comment by rudeboybert** [comment]
Done. thanks

**Comment by rudeboybert** [comment]
Hi @Ruxton I removed `order_online_url` a while back. Are there any other problems?

**Comment by pkq** [comment]
Looks good @rudeboybert! cc @ismayc

**Pull request merged**